### PR TITLE
SDIT-1825 Basic physical attributes migration

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonMigrationService.kt [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    }
+    // TODO create mappings
+    telemetryClient.trackEvent(
+      "prisonperson-migration-entity-migrated",
+      mapOf(
+        "offenderNo" to offenderNo,
+        "migrationId" to context.migrationId,
+      ),
+    )
+  }
```
Comment (mikehalmamoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonMigrationService.kt:86]: Currently just done enough to get the basic migration working - needs fleshing out to create mappings, test complicated height/weight histories, test telemetry etc.

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonMigrationService.kt [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  override suspend fun migrateNomisEntity(context: MigrationContext<PrisonerId>) {
+    log.info("attempting to migrate ${context.body}")
+    val offenderNo = context.body.offenderNo
+
+    val physicalAttributes = prisonPersonNomisApiService.getPhysicalAttributes(offenderNo)
+    val dpsIds = mutableListOf<Long>()
+    physicalAttributes.bookings.forEach { booking ->
+      booking.physicalAttributes.forEach { pa ->
+        val (lastModifiedAt, lastModifiedBy) = pa.lastModified()
+        prisonPersonDpsService.migratePhysicalAttributes(
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonMigrationService.kt:65]: Not convinced this is a good idea, so for each prisoner you will be called DPS several times, what do you do if the sat the 6th call fails, won't it migrate the first 5 all over again and the mappings will be screwed up?

**Review by andymarke — APPROVED** [review]
LGTM

**Review by andymarke — APPROVED** [review]
lgtm

**Pull request merged**